features/order_confirmation.feature

```diff
@@ -0,0 +1,12 @@
+Feature: As a Visitor.
+  In order to know when my order is ready for pick-up.
+  After clicking buy now, I would like to see an order confirmation with estimated pick up time.
+
+  Background:
+  Given I have a dish in my order and I am on the checkout_screen
+
+  Scenario: Visitor clicks Buy now! and is taken to a order confirmation page with pick-up time.
+  When I click "Buy now!"
+  Then I should be on the order confirmation page
+  And I should see "Thank you for your order! Your food will be ready for pick up at"
+  And I should see a pick up time 30 minutes from now
```

Comment:
The variable is not stored as constant=30 in this feature description and therefore not passed as constant=30 to the steps def. 

However it is hardcoded into the step def as 1800. 

Should we pass it as a variable from the feature to the step def as a variable do you think?